stylesheets/main.css

```diff
@@ -369,6 +369,537 @@
+  font-weight: normal;
+}
+
+@font-face {
+  font-family: 'Playfair Display';
+  src: url("../assets/playfair-display-v30-vietnamese_latin-ext_cyrillic-italic.woff") format("woff");
+  font-weight: normal;
+  font-style: italic;
+}
+
+@font-face {
+  font-family: 'Playfair Display';
+  src: url("../assets/playfair-display-v30-vietnamese_latin-ext_cyrillic-700.woff") format("woff");
+  font-weight: bold;
+}
+
+@font-face {
+  font-family: 'Playfair Display';
+  src: url("../assets/playfair-display-v30-vietnamese_latin-ext_cyrillic-700italic.woff") format("woff");
+  font-weight: bold;
+  font-style: italic;
+}
+
+@font-face {
+  font-family: 'PT Sans';
+  src: url("../assets/pt-sans-v17-latin-ext_cyrillic-ext_cyrillic-regular.woff") format("woff");
+  font-weight: normal;
+}
+
+@font-face {
+  font-family: 'PT Sans';
+  src: url("../assets/pt-sans-v17-latin-ext_cyrillic-ext_cyrillic-italic.woff") format("woff");
+  font-weight: normal;
+  font-style: italic;
+}
+
+@font-face {
+  font-family: 'PT Sans';
+  src: url("../assets/pt-sans-v17-latin-ext_cyrillic-ext_cyrillic-700.woff") format("woff");
+  font-weight: bold;
+}
+
+@font-face {
+  font-family: 'PT Sans';
+  src: url("../assets/pt-sans-v17-latin-ext_cyrillic-ext_cyrillic-700italic.woff") format("woff");
+  font-weight: bold;
+  font-style: italic;
+}
+
+@font-face {
+  font-family: 'PT Serif';
+  src: url("../assets/pt-serif-v17-latin-ext_cyrillic-ext_cyrillic-regular.woff") format("woff");
+  font-weight: normal;
+}
+
+@font-face {
+  font-family: 'PT Serif';
+  src: url("../assets/pt-serif-v17-latin-ext_cyrillic-ext_cyrillic-italic.woff") format("woff");
+  font-weight: normal;
+  font-style: italic;
+}
+
+@font-face {
+  font-family: 'PT Serif';
+  src: url("../assets/pt-serif-v17-latin-ext_cyrillic-ext_cyrillic-700.woff") format("woff");
+  font-weight: bold;
+}
+
+@font-face {
+  font-family: 'PT Serif';
+  src: url("../assets/pt-serif-v17-latin-ext_cyrillic-ext_cyrillic-700italic.woff") format("woff");
+  font-weight: bold;
+  font-style: italic;
+}
+
+@font-face {
+  font-family: 'Raleway';
+  src: url("../assets/raleway-v28-vietnamese_latin-ext_cyrillic-ext_cyrillic-regular.woff") format("woff");
+  font-weight: normal;
+}
+
+@font-face {
+  font-family: 'Raleway';
+  src: url("../assets/raleway-v28-vietnamese_latin-ext_cyrillic-ext_cyrillic-italic.woff") format("woff");
+  font-weight: normal;
+  font-style: italic;
+}
+
+@font-face {
+  font-family: 'Raleway';
+  src: url("../assets/raleway-v28-vietnamese_latin-ext_cyrillic-ext_cyrillic-700.woff") format("woff");
+  font-weight: bold;
+}
+
+@font-face {
+  font-family: 'Raleway';
+  src: url("../assets/raleway-v28-vietnamese_latin-ext_cyrillic-ext_cyrillic-700italic.woff") format("woff");
+  font-weight: bold;
+  font-style: italic;
+}
+
+@font-face {
+  font-family: 'Roboto';
+  src: url("../assets/roboto-v30-vietnamese_latin-ext_greek-ext_greek_cyrillic-ext_cyrillic-300.woff") format("woff");
```

Comment:
Roboto 300 and 500 rules are removed form source (.scss) files but are still present in style.css.

Please run `grunt`.